### DISCUSSION TITLE
Want to collaborate with the Spanish translation

### DISCUSSION THREAD

**Discussion by consultorseovalencia** [pop-os/shell#1405] [Q&A]
I have recently collaborated with the spanish translation of Pop Desktop Settings Widget and I would also like to provide the translation for Pop Shell. 

The problem is that there is no i18n folder with an ¨en¨ folder and the corresponding .ftl file to start working. Can someone help me with this? 

Thank you.

**Comment by ghost** [comment]
By skimming through the code and docs, I haven't found anything on localization, it seems to me it's currently not possible. I even tried searching for some keywords to find hints in the project, but with no success. 

You could try double checking at https://chat.pop-os.org/pop-os/channels/development.

Hope it helps ^^

**Reply by mmstick** [reply]
There isn't any support for it at this time, but I'm open to PRs adding it
